xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2358,24 +2358,24 @@ xe.admin.passwordReset.step2.title=Reset your password
 xe.admin.passwordReset.step2.newPassword.label=New password
 xe.admin.passwordReset.step2.newPasswordVerification.label=Re-enter new passwod
 xe.admin.passwordReset.step2.submit=Save
-xe.admin.passwordReset.step2.success=The password has been successfully set.
+xe.admin.passwordReset.step2.success=The password has been successfully reset.
```

Comment:
You changed the meaning of the message. This is displayed after you set a new password.

Reply:
Actually the above would have been shown during creating new password so I thought it that way....

Reply:
The step 1 is sending you a mail to change the password, then in the step 2 you set a new password.